week-1/the-pokes/tictactoe.rb

```diff
@@ -0,0 +1,119 @@
+puts "---- Bienvenidos a Tic Tac Toe Codeable ----"
+puts "Reglas del juego:"
+puts "--------------------------------------------"
+puts "1. Los jugadores eligiran la posicion donde
+  marcaran"
+puts "--- Tablero ---"
+
+puts "\n 1 | 2 | 3 "
+puts "-----------"
+puts " 4 | 5 | 6 "
+puts "-----------"
+puts " 7 | 8 | 9 "
+
+puts "\nEmpecemos!!!"
+
+puts "\n   |   |   "
+puts "-----------"
+puts "   |   |   "
+puts "-----------"
+puts "   |   |   "
+
+COMBINATIONS_WIN = [
+  [1,2,3],  
+  [4,5,6],
+  [7,8,9],
+  [1,4,7],
+  [2,5,8],
+  [3,6,9],
+  [1,5,9],
+  [3,5,7]
+]
+
+TABLE = [
+  1,
+  2,
+  3,
+  4,
+  5,
+  6,
+  7,
+  8,
+  9
+]
+
+def notPlayed(value)
+  arrNP = TABLE - value
+  return arrNP 
+end
+
+def turnHuman(notplayed)
+    value = 0
+    loop do
+      puts "\nEs el turno del Humano"
+      value = gets.chomp.to_i
+      break if ( ( notplayed.include?(value)) && (value >= 1) && (value <=9) )
+    end
+    return value
+end
+
+def turnBot(value)
+  if(value != [])
+    value.sample
+  end
+end
+
+def tictactoe()
+  
+  value_human = []
+  value_bot = []
+
+  band = false
+
+  player = "X"
+  bot = "O"
+
+  table = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+
+  while (band == false) do
+
+
+    # pos_human : Es el valor que ingresa el usuario
+    pos_human = turnHuman(notPlayed(value_human+value_bot))
+    table[pos_human-1] = player
+    value_human.push(pos_human)
+    
+
+    pos_bot = turnBot(notPlayed(value_human+value_bot))
+    table[pos_bot-1] = bot
+    value_bot.push(pos_bot)
+
+    # player == "X" ? value_human.push(pos) : value_bot.push(pos)
+
+    puts "\n #{table[0]} | #{table[1]} | #{table[2]} "
+    puts "-----------"
+    puts " #{table[3]} | #{table[4]} | #{table[5]} "
+    puts "-----------"
+    puts " #{table[6]} | #{table[7]} | #{table[8]} "
+
+    puts "Soy un humna #{value_human}"
```

Comment:
You have a typo here in `humna`
```suggestion
    puts "Soy un humano #{value_human}"
```

Reply:
Ok thanks,  we will correct.